shared/src/main/scala/squants/QuantityRange.scala

```diff
@@ -21,7 +21,9 @@ import scala.annotation.tailrec
  * @tparam A the Quantity Type
  */
 case class QuantityRange[A <: Quantity[A]](lower: A, upper: A) {
-  if (lower >= upper) throw new IllegalArgumentException("QuantityRange upper bound must be greater than or equal to the lower bound")
+  if (lower >= upper) {
```

Comment:
It would. We can add alternate builder functions. We can do that for 1.3 if you want, or hold off until afterwards.